admin/helpers/index.js

```diff
@@ -58,7 +57,7 @@ module.exports = async function (app) {
   app.locals.crownCopyrightMessage = null
   app.locals.googleTrackingId = config.GOOGLE_TRACKING_ID
   app.locals.appInsightsClientKey = config.Monitoring.ApplicationInsights.Key
-  app.locals.deployVersion = appVersion
+  app.locals.deployVersion = buildNumber
```

Comment:
hmm do we need both locals if they have the same value? 🤔

Reply:
i think buildNumber is a vso generated item, which we inject into build.txt during `dev build`
this, in my humble opinion, is a more useful value as it maps to a build, and therefore a commit